.github/workflows/cuda-ssh.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - "cuda-ssh/**"
+      - ".github/workflows/cuda-ssh.yml"
+      - ".github/workflows/build.yml"
+
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        tag:
+          - ceeb8c2-base-cuda11.8.0-torch2.0.1-vision0.15.2-audio2.0.2
```

Comment:
This almost belongs in the ml containers repo, as this would let us update to CI. Hm. Not a blocker, though.